src/components/TxData.js

```diff
@@ -90,29 +102,23 @@ class TxData extends React.Component {
   /**
    * Checks if token was already added and if it's a known token, then add it
    *
-   * @param {Array} tokens Array of already added tokens
    * @param {number} tokenData Represents the index of the token in this transaction
+   * @return {Object} Token config object with {uid, name, symbol}
    */
-  checkToken = (tokens, tokenData) => {
-    if (tokenData === HATHOR_TOKEN_INDEX) {
+  checkToken = (tokenData) => {
+    if (tokenData === hathorLib.constants.HATHOR_TOKEN_INDEX) {
       return;
```

Comment:
What is the return type of this method? It seems that there are some `return` along the method, but the last line is a `return tokenConfig`. Maybe we should `return null` here?

Reply:
Makes sense, added it.